src/Pass/ImgTagTransformPass.php

```diff
@@ -110,6 +201,19 @@ protected function getImageWidthHeight($src)
 
         // Try obtaining image size without having to download the whole image
         $size = $this->fastimage->getImageSize($img_url);
```

Comment:
@mpatnode-si , in the url generation one can specify the size that he wants to get but that's not always what actual size is  ( if you crop the image, then that's the size 100% of the time), thus makes the size extraction from the url unreliable.